packages/react-scripts/config/webpack.config.js

```diff
@@ -37,6 +37,10 @@ const typescriptFormatter = require('react-dev-utils/typescriptFormatter');
 const getCacheIdentifier = require('react-dev-utils/getCacheIdentifier');
 // @remove-on-eject-end
 const postcssNormalize = require('postcss-normalize');
+// CorberWebpackPlugin for corber hot-reloading
+const CorberWebpackPlugin = require('corber-webpack-plugin');
+// Included to manually append corber assets
+const HtmlWebpackTagsPlugin = require('html-webpack-tags-plugin');
```

Comment:
@trayldev do we need this plugin added to a `package.json` or `yarn.lock` file?

Reply:
Moving from devDepen to dependencies.

Reply:
Adding.